tt_metal/common/metal_soc_descriptor.cpp

```diff
@@ -109,10 +109,22 @@ CoreCoord metal_SocDescriptor::get_physical_tensix_core_from_logical(const CoreC
     return physical_tensix_core;
 }
 
+CoreCoord metal_SocDescriptor::get_physical_dram_core_from_logical(const CoreCoord &logical_coord) const {
+    TT_FATAL(
+        (logical_coord.x < this->get_num_dram_channels()) and
+        (logical_coord.y == 0),
+        "Bounds-Error -- Logical_core={} is outside of logical_grid_size={}",
+        logical_coord.str(),
+        CoreCoord(1, this->get_num_dram_channels())
+    );
+    return this->get_preferred_worker_core_for_dram_channel(logical_coord.x);
```

Comment:
ah so logical DRAM coordinates are (channel, 0)?

Reply:
Yes, it's just a linear 1d mapping of the channel. Wasn't sure if it mattered if it's x or y, or if we should have some sort of 2d mapping to better reflect the physical location of the dram cores so that users can optimize placement more.